scripts/scripts.js

```diff
@@ -239,6 +240,7 @@ export async function load404() {
 async function loadEager(doc) {
   setPageLanguage();
   decorateTemplateAndTheme();
+  fetchLanguageNavigation(`/${getLanguage()}`);
```

Comment:
@rrusher  why It's important to call navigation in eager?

Reply:
@mkbansal1 I believe by the time header loads navigation items will be ready. Now, weather to load it in lazy vs eager might be debatable. If loading it in eager isn't impacting the score (which I think it isn't looking at the score), then I think it is fine.